studio/schemas/resource.js

```diff
@@ -0,0 +1,249 @@
+import {MdKitchen as icon} from 'react-icons/md'
+
+export default {
+  name: 'resource',
```

Comment:
this is the main thing, it's a resource and can be used to describe a lot of different content type things that you'd find on egghead etc